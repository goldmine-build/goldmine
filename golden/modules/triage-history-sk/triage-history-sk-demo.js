import './index';

Date.now = () => new Date('2020-03-10T01:01:01-04:00');

const ele = document.querySelector('#with_data');
ele.history = [
  {
    user: 'test@example.com',
    ts: '2020-03-09T12:56:08.000000000-04:00',
  },
];
