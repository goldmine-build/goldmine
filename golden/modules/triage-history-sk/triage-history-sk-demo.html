<!DOCTYPE html>
<html>
<head>
  <title>triage-history-sk</title>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>
  <h2>Draws nothing when empty</h2>
  <triage-history-sk id="empty"></triage-history-sk>

  <h2>Shows the last entry</h2>
  <triage-history-sk id="with_data"></triage-history-sk>
</body>
</html>
